### DISCUSSION TITLE
In tests row.value is undefined

### DISCUSSION THREAD

**Discussion by anmatika** [TanStack/table#2769] [General]
Has anyone tested an app which uses react-table? I cannot make it render data. It render cells but cell values are undefined.
Testing an app which uses react-table by react-testing-library. In normal use rows.values gets populated. But with the same data they become undefined.

```
render(
    <ThemeProvider>
      <ProjectTable
        columns={columns}
        data={data}
        onChangeSort={() => {}}
        pageCount={1}
        fetchData={() => {}}
        initialPageSize={200}
      />
```
tableInstance:

![image](https://user-images.githubusercontent.com/1430995/95467707-72ba1f00-0986-11eb-8fb7-fbd0198be38a.png)

**Comment by anmatika** [comment]
It works now. Renders just fine in the test.
My bad. I had just the data in columns but forgot to add `accessor` function into the columns.

**Comment by murageh** [comment]
Hi, i've been trying to write a test on an app that uses react-table and it fails with the message
"Cannot find module 'react' from '../node_modules/react-table/dist/react-table.development.js'

any ideas? or how did you do it?
